quick-start-guide/src/main/resources/QSG/bin/qsg.sh

```diff
@@ -110,8 +110,9 @@ echo "|  So choose your desired approach from the list below to enable the  |"
 echo "|  required settings.                                                 |"
 echo "|                                                                     |"
 echo "|    Press 1 - Enable Self User Registration(without any config.)     |"
-echo "|               [ This will enable self signup in the IS without any  |"
-echo "|               other configuration changes.]                         |"
+echo "|               [ This will enable self sign-up in the IS without any  |"
+echo "|               other configuration changes. But although you see the |"
+echo "|               notification in UI, you will not get email.]          |"
```

Comment:
Instead of mention that "But although you see the notification in UI, you will not get email", can't we fix it in the code level to not to show this wrong notification in the UI?